lib/util/regionList.ts

```diff
@@ -0,0 +1,34 @@
+import { Region } from "./region";
+import { PolusBuffer } from "./polusBuffer";
+
+export class RegionList {
+  type: number;
```

Comment:
Maybe an enum here for the two values?
```ts
export enum ServerType {
	Custom = 0,
	Official = 1
}
```
Not sure if 2 value enums are good, @SomeoneWeird thoughts?

Reply:
The first `int32` is not a type, but instead the index of the selected server from the servers array

Reply:
gotcha, i'll write a select server method then, perhaps, to set this value?

Reply:
Just saw the change on weewoo wiki, yeah you're right @codyphobe